File: resources/expected/ros2/fan_in_fan_out/strict/src/fan_in_fan_out_system_cpp_pkg/src/base_code/fanIn_consumer_runner.cpp
#include "fan_in_fan_out_system_cpp_pkg/user_headers/fanIn_consumer_src.hpp"

//========================================================
// Re-running Codegen will overwrite changes to this file
//========================================================

fanIn_consumer::fanIn_consumer() : fanIn_consumer_base()
{
    // Invoke initialize entry point
    initialize();

    PRINT_INFO("fanIn_consumer infrastructure set up");
}

int main(int argc, char **argv)
{
    rclcpp::init(argc, argv);
    auto executor = rclcpp::executors::MultiThreadedExecutor();
    auto node = std::make_shared<fanIn_consumer>();
    executor.add_node(node);
    executor.spin();
    rclcpp::shutdown();
    return 0;
}

File: resources/expected/ros2/python-fan_in_fan_out/lax/src/fan_in_fan_out_system_py_pkg/fan_in_fan_out_system_py_pkg/base_code/fanIn_consumer_runner.py
#!/usr/bin/env python3
import rclpy
from rclpy.node import Node
from rclpy.executors import MultiThreadedExecutor
from fan_in_fan_out_system_py_pkg.user_code.fanIn_consumer_src import initialize
from fan_in_fan_out_system_py_pkg.base_code.fanIn_consumer_base_src import fanIn_consumer_base
#========================================================
# Re-running Codegen will overwrite changes to this file
#========================================================

class fanIn_consumer(fanIn_consumer_base):
    def __init__(self):
        super().__init__()
        # invoke initialize entry point
        initialize(self)

        self.get_logger().info("fanIn_consumer infrastructure set up")

def main(args=None):
    rclpy.init(args=args)
    node = fanIn_consumer()
    executor = MultiThreadedExecutor()
    executor.add_node(node)
    executor.spin()
    rclpy.shutdown()

if __name__ == "__main__":
    main()
